lamindb/core/_sync_git.py

```diff
@@ -24,8 +24,7 @@ def get_git_repo_from_remote() -> Path:
         f"running outside of synched git repo, cloning {repo_url} into {repo_dir}"
     )
     result = subprocess.run(
-        f"git clone --depth 10 {repo_url}.git",
-        shell=True,
+        ["git", "clone", "--depth", "10", f"{repo_url}.git"],
```

Comment:
I'm pretty sure the `shell=True` is there for a purpose. 🤔 

I have to admit that I never attempted to understand when it's necessary and when not; but I remember that some things didn't work without it. I'd be astonished if this works without it (meaning I added it "just for fun").

Is it possible that the Mac environment is at fault and that's why this might not be caught by CI? I will see if I can still locally run the tests for this.

Reply:
For `shell=False` to work, we need to pass the command as a list. The subprocess module then knows what the command is and what the arguments are.

But ya, we can unpack and split them.